app/views/compute_resources_vms/form/_network.html.erb

```diff
@@ -1,12 +1,22 @@
-<div class="fields">
-  <%
-     nat = compute_resource.networks
-  -%>
+<script>
```

Comment:
I'd suggest putting this in a dedicated JS file.
You can then use something like this to trigger the methods.

```javascript
$(document).on('ContentLoad', populateNetworks);
```